app/controllers/v0/post911_gi_bill_statuses_controller.rb

```diff
@@ -31,8 +31,13 @@ def render_error_json(error_type)
         # 503
         raise EVSS::GiBillStatus::ServiceException
       when EVSS::GiBillStatus::GiBillStatusResponse::KNOWN_ERRORS[:vet_not_found]
-        # 404
-        raise Common::Exceptions::RecordNotFound, @current_user.email
+        begin
+          GibsNotFoundUser.log(@current_user)
+        rescue => e
+          log_exception_to_sentry(e)
```

Comment:
How is Sentry used in regards to this error? Is there additional context we need to reference regularly, or can we consider this a non-exception (since it's a known condition) and set up an alert with prometheus if needed over `api_evss_gi_bill_status_fail{error="vet_not_found"}`?

Reply:
most of the time it's a known condition, I think I can reduce the frequency by splitting out activerecord errors and not sending those to sentry.